internal/services/v1/relationships.go

```diff
@@ -290,13 +325,18 @@ func (ps *permissionServer) DeleteRelationships(ctx context.Context, req *v1.Del
 			return err
 		}
 
+		if len(deleteMutations) > 0 {
+			return rwt.WriteRelationships(ctx, deleteMutations)
```

Comment:
can you please add an OpenTelemetry log entry to the trace indicating the number of elements to be removed? Or is that visible already downstream via `WriteRelationships`?

Reply:
It should be in the downstream call in the datastore, yes